plugin_pbs.py

```diff
@@ -84,15 +85,29 @@ def _read_serialised_qstatq( fn, write_method=options.write_method):
         qstatq_list = []
         logging.debug("Opening %s" % fn)
         with open(fn, 'r') as fin:
-            qstatqs_total = (write_method.endswith('yaml')) and yaml.load_all(fin) or json.load(fin)
-            for qstatq in qstatqs_total:
-                qstatq_list.append(qstatq)
-            total = qstatq_list.pop()
-            total_running_jobs, total_queued_jobs = total['Total_running'], total['Total_queued']
-        return int(eval(total_running_jobs)), int(eval(total_queued_jobs)), qstatq_list
+            if write_method.endswith('yaml'):
+                qstatqs_total = yaml.load_all(fin)
+            else:
+                qstatqs_total = json.load(fin)
+
```

Comment:
I'm weeping because of this, but the and/or trick doesn't work anymore if yaml.load_all(fin) can evaluate to False